style.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.item {
```

Comment:
.item and .item2 might be named differently (more intuitive as to what they are doing in the code)